assets/js/blocks/collection-filters/inner-blocks/active-filters/frontend.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { store, navigate, getContext } from '@woocommerce/interactivity';
+
+type ActiveFiltersContext = {
+	queryId: number;
+	params: string[];
+};
+
+store( 'woocommerce/collection-active-filters', {
+	actions: {
+		clearAll: () => {
+			const { params } = getContext< ActiveFiltersContext >();
+			const url = new URL( window.location.href );
+			const { searchParams } = url;
+
+			params.forEach( ( param ) => searchParams.delete( param ) );
```

Comment:
I noticed there is a bug here it will remove params such as post ID and will navigate away from a product for example. you'll just need to whitelist a list of all the product filters for now probably?